modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/notifications/DLExpiredEntryUserNotificationDefinition.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.document.library.web.internal.notifications;
+
+import com.liferay.document.library.constants.DLPortletKeys;
+import com.liferay.portal.kernel.model.UserNotificationDeliveryConstants;
+import com.liferay.portal.kernel.notifications.UserNotificationDefinition;
+import com.liferay.portal.kernel.notifications.UserNotificationDeliveryType;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Sergio González
```

Comment:
Can you remove Sergio from here? I don't think he wants to be involved in this :-)

Reply:
XD